public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -45,11 +45,14 @@ import { from, empty, Observable } from 'rxjs';
 import { delay, expand, map, mergeMap, tap, finalize, catchError } from 'rxjs/operators';
 import { CloudWatchLanguageProvider } from './language_provider';
 
-const TSDB_QUERY_ENDPOINT = '/api/tsdb/query';
 import { VariableWithMultiSupport } from 'app/features/templating/types';
 import { RowContextOptions } from '@grafana/ui/src/components/Logs/LogRowContextProvider';
 import { AwsUrl, encodeUrl } from './aws_url';
 
+const TSDB_QUERY_ENDPOINT = '/api/tsdb/query';
+const LOG_IDENTIFIER_INTERNAL = '__log_grafana_internal__';
```

Comment:
hmm same nitpick applies here and a bit unfortunate we need to duplicate it. Please add a comment pointing to the Go code where this is also defined and vice versa.